.github/workflows/msbuild.yml

```diff
@@ -55,13 +61,22 @@ jobs:
       run: pip install lit OutputCheck pyyaml
     - uses: actions/setup-node@v1
     - run: npm install bignumber.js
-    - name: Create release
-      run: python dafny/Scripts/package.py 0.0.0-CI --os ubuntu --skip_manual=true
-    - name: Unpack release
-      uses: TonyBogdanov/zip@1.0
+    - name: Checkout Dafny
+      uses: actions/checkout@v2
       with:
-        args: unzip -qq dafny/Package/dafny-0.0.0-CI-x64-ubuntu-16.04.zip -d unzippedRelease
-    - name: Run lit tests
+        submodules: recursive
+        path: dafny
+    - name: Create release
+      run: |
+        python dafny/Scripts/package.py 0.0.0-CI --os ${{ matrix.osb }} --skip_manual=true --trial=true
```

Comment:
Consider removing the `skip_manual` option and skipping the manual building when trial is true.

Reply:
trial does intend to exercise the whole build process, including the manual, just not be concerned about updating the version numbers; so I think keeping the two options independent is better.